product_docs/docs/epas/15/epas_guide/03_database_administration/02_index_advisor/02_index_advisor_configuration.mdx

```diff
@@ -23,19 +23,17 @@ When selecting a storage schema for the Index Advisor table, function, and view,
 2.  Create the `index_advisor` extension, which creates the database objects. Log in as superuser and use the command:
 
     ```sql
-    edb=# create extension index_advisor ;
-    __OUTPUT__
-    CREATE EXTENSION
+    # running as superuser
```

Comment:
We need to make this more clear, as this superuser is a database superuser connecting to psql and running this command